packages/cursorless-vscode/src/ide/vscode/VscodeOpenLink.ts

```diff
@@ -17,7 +19,10 @@ export default async function vscodeOpenLink(
   }
 
   if (filteredLinks.length === 0) {
-    return false;
+    let commandId = options?.openInSplit
+      ? "editor.action.revealDefinitionAside"
+      : "editor.action.revealDefinition";
+    await vscode.commands.executeCommand(commandId);
```

Comment:
Also you still need a return here. Otherwise you will run open link a few lines below on undefined.